packages/plugin-core/src/survey.ts

```diff
@@ -772,15 +812,17 @@ export class SurveyUtils {
           const onboardingSurvey = InactiveUserOnboardingSurvey.create();
           const additionalCommentSurvey =
             InactiveUserAdditionalCommentSurvey.create();
+          const feedbackCallSurvey = InactiveUserFeedbackCallSurvey.create();
           const discordPlugSurvey = InactiveUserPlugDiscordSurvey.create();
 
-          const reasonResults = await reasonSurvey.show(1, 4);
-          const onboardingResults = await onboardingSurvey.show(2, 4);
+          const reasonResults = await reasonSurvey.show(1, 5);
+          const onboardingResults = await onboardingSurvey.show(2, 5);
           const additionCommentResult = await additionalCommentSurvey.show(
             3,
-            4
+            5
           );
-          const discordPlugResult = await discordPlugSurvey.show(4, 4);
+          const feedbackCallResults = await feedbackCallSurvey.show(4, 5);
```

Comment:
I think the feedback call should only be shown if the user refused the onboarding survey. I found it a little confusing otherwise: I said yes to meet for the onboarding, and then it also asked if I want to meet again.

Reply:
That makes sense. Also, the chain of questions are mostly reworded questions that I took from the lapsed user survey, and I think it would make sense to go straight to the point and ask if they want to book a feedback session.

We are giving up on the existing answers, but we aren't getting a lot of that right now. something to consider.

At the very least I agree it's a bit confusing right now and your suggestion makes sense.